arch/arm/Kconfig

```diff
@@ -155,12 +155,12 @@ config CPU_HAS_ARM_MPU
 	  This option is enabled when the CPU has a Memory Protection Unit (MPU)
 	  in ARM flavor.
 
-config CPU_HAS_NXP_MPU
+config CPU_HAS_NXP_SYSMPU
 	bool
 	select CPU_HAS_MPU
 	help
-	  This option is enabled when the CPU has a Memory Protection Unit (MPU)
-	  in NXP flavor.
+	  This option is enabled when the CPU has an NXP System Memory Protection
```

Comment:
Replace `an` with `a`

Reply:
nitpicking: I'm not a native English speaker, so I may be wrong, but I believe `an` is correct here because `en-ex-pee` starts with a vowel sound, regardless of whether the first letter is a vowel

Reply:
> nitpicking: I'm not a native English speaker, so I may be wrong, but I believe `an` is correct here because `en-ex-pee` starts with a vowel sound, regardless of whether the first letter is a vowel

Yup, that's correct :)